cabal.project

```diff
@@ -24,6 +24,10 @@ index-state:
 
 packages: ./.
 
+-- Required to work around the contra-tracer/iog-contra-tracer mess.
```

Comment:
Can we expand on the mess here. What does the upper bound here do, and how does it help? When can we remove it?

Reply:
There is the mess, but more importantly the API has changed I  don't think now is the right time to deal with that.